docs/source/pages/using.rst

```diff
@@ -218,8 +262,13 @@ Running the policy engine
 Macaron's policy engine accepts policies specified in `Datalog <https://en.wikipedia.org/wiki/Datalog>`_. An example policy
 can verify if a project and all its dependencies pass certain checks. We use `Soufflé <https://souffle-lang.github.io/index.html>`_
 as the Datalog engine in Macaron. Once you run the checks on a target project as described :ref:`here <analyze-action>`,
-the check results will be stored in ``macaron.db`` in the output directory. We can pass the check results to the policy
-engine and provide a Datalog policy file to be enforced by the policy engine.
+the check results will be stored in ``macaron.db`` in the output directory. Because the check results of different software components can be stored in the database at ``macaron.db``, we must specify the target software component in the Datalog policy file to be enforced by the policy engine. There are two ways that we can
+specify the target software component in the Datalog policy file:
```

Comment:
```suggestion
the check results will be stored in ``macaron.db`` in the output directory. We pass the check results to the policy engine by providing the path to ``macaron.db`` together with a Datalog policy file to be validated by the policy engine. In the Datalog policy file, we must specify the identifier for the target software component that we are interested in to validate the policy against. These are two ways to specify the target software component in the Datalog policy file:
```

Reply:
Fixed in https://github.com/oracle/macaron/pull/401/commits/229e0394f00c7f257f46987c5549b0d72143aa2a